web/thesauruses/python/3/queues_stacks.json

```diff
@@ -162,48 +160,48 @@
       "double_ended_queue_data_structure": {
         "name": "Data structure that backs the simple queue",
         "code": [
-          "list"
+          "linear data structure"
         ]
       },
       "stack_import_statement": {
         "name": "Import statement to add stacks",
-        "not-implemented": "true"
+        "code": [
+          "from queue import LifoQueue"
+        ]
 
       },
       "stack_data_type": {
         "name": "Stack data type name",
         "code": [
-          "list"
+          "queue.LifoQueue"
         ]
       },
       "stack_create_statement": {
         "name": "Create a stack",
         "code": [
-          "stack = ['One','Two','Three']"
+          "queue = LifoQueue()"
         ]
       },
       "stack_create_copy_statement": {
         "name": "Create a stack from existing stack",
         "code": [
-          "new_stack = stack"
+          "new_queue = queue"
         ]
       },
       "stack_create_copy_from_list_statement": {
         "name": "Create a stack from a list",
-        "code": [
-          "mylist = ['One','Two','Three']\nstack = mylist"
-        ]
+        "not-implemented": "true"
       },
       "stack_destroy_statement": {
         "name": "Destroy/delete a stack",
         "code": [
-          "stack.clear()"
+          "del(queue)"
```

Comment:
Should this be `del(stack)` ?

Reply:
I think this accidentally got replaced when i was replacing the other instances, I'll have this resolved as this doesn't fit in.

Please let me know if you want me to use the "stack" variable name for the stack category or should i use queue with all of them.

Reply:
It could also be `del(lifoqueue)` since that's what a stack is.